<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game_over</class>
 <widget class="QDialog" name="Game_over">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="retouraumenu">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
     border-width: 6px;
     border-color: grey;
background-color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>Retour au menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>550</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="mice.qrc">:/images/game_over.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="rejouer">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
     border-width: 6px;
     border-color: grey;
background-color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>Rejouer</string>
   </property>
  </widget>
  <widget class="QLabel" name="first_player">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SWRomnt</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(212, 212, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="segundo_playado">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SWRomnt</family>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127);
color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="dritter_spieler">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SWRomnt</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SWRomnt</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
color: rgb(212, 212, 0);</string>
   </property>
   <property name="text">
    <string>1st</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SWRomnt</family>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>2nd</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SWRomnt</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 127);</string>
   </property>
   <property name="text">
    <string>3rd </string>
   </property>
  </widget>
  <widget class="QLabel" name="segundo_playado_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>290</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SWRomnt</family>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127);
color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="dritter_spieler_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SWRomnt</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="first_player_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>230</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SWRomnt</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(212, 212, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>rejouer</zorder>
  <zorder>retouraumenu</zorder>
  <zorder>first_player</zorder>
  <zorder>segundo_playado</zorder>
  <zorder>dritter_spieler</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>segundo_playado_2</zorder>
  <zorder>dritter_spieler_2</zorder>
  <zorder>first_player_2</zorder>
 </widget>
 <resources>
  <include location="mice.qrc"/>
 </resources>
 <connections/>
</ui>
